src/herder/TxSetFrame.cpp

```diff
@@ -149,7 +149,8 @@ struct SurgeSorter
         if (tx1->getSourceID() == tx2->getSourceID())
             return tx1->getSeqNum() < tx2->getSeqNum();
 
-        // disregard fees when sorting whitelisted txs
+        // whitelisted txs are not charged fees, so disregard them when
+		// sorting whitelisted txs
```

Comment:
cosmetic but still - indentation is off because it's using tabs instead of space. see if you can change your editor to match the code standard, and apply to rest of the code - this should fix all spacing issues in the pr